NAMESPACE

```diff
@@ -157,6 +185,8 @@ import(stats)
 importFrom(dplyr,"%>%")
 importFrom(dplyr,arrange)
 importFrom(dplyr,count)
+importFrom(dplyr,everything)
```

Comment:
This is a tidyselect function which dplyr reexports: https://tidyselect.r-lib.org/reference/everything.html

Reply:
switched to tidyselect::everything and pushed so will mark this as resolved